docs/user-guides/oauth-configuration-helpers.md

```diff
@@ -46,6 +46,27 @@ authentication:
           scopes: ["https://graph.microsoft.com/.default"] # For Client Credentials
 ```
 
+#### Keycloak no TLS
+
+An example of how to configure oauth authentication for keycloak provider using no TLS configuration. To run in a development configuration for instance, when your Keycloak server doesn't have certificates signed by known authority.
+
+```yaml
+authentication:
+  providers:
+    oauth:
```

Comment:
Did you try with oidc? I believe that if oidc works, giving just the issuer URL would be simpler than all the URLs one by one